mplib/planner.py

```diff
@@ -66,9 +68,8 @@ def __init__(
         self.robot = ArticulatedModel(
             self.urdf,
             self.srdf,
-            [0, 0, -9.81],
-            user_link_names,
-            user_joint_names,
+            link_names=user_link_names,
+            joint_names=user_joint_names,
             convex=kwargs.get("convex", False),
```

Comment:
That was the change I was referring to. We should probably let the user know that switching this to True will increase performance

Reply:
You mean the `convex` argument, right?
Yes that's true. Actually I think we should just make it default to `True` and let user worry about nonconvex mesh themselves?
I think if you're working with robot collision meshes, every simulator I know out there will just use the convex-hull of the given mesh.

Reply:
Actually, can you try with nonconvex meshes and see if the method call will throw an error like we found last Friday?
You can test for mesh convexness with the `is_watertight()` method from `trimesh`.
Convex <=> watertight and vice versa.

Reply:
Confirmed that `convex=True` will throw error if using non-convex meshes.